include/javabind/optional.hpp

```diff
@@ -19,8 +19,6 @@ namespace javabind
     struct ClassTraits<std::optional<T>>
     {
         constexpr static std::string_view class_name = arg_type_t<T>::class_name;
-        constexpr static std::string_view class_path = arg_type_t<T>::class_path;
-        constexpr static std::string_view java_name = arg_type_t<T>::java_name;
```

Comment:
I would recommend that we also extend `javabind.cpp` and the unit tests such that the [GitHub Actions workflow](https://github.com/hunyadi/javabind/actions) can catch this error as well. (Currently, it passes for both Linux and Windows on `master`.) If I introduce the following in `javabind.cpp`, I can reproduce the error:
```
// optional
.function<StaticSample::pass_optional<Rectangle>>("pass_optional_rectangle")
.function<StaticSample::pass_optional<int>>("pass_optional_int")
.function<StaticSample::pass_optional<std::string>>("pass_optional_string")
;
```

Reply:
Done